OKR/PROJECT.md

```diff
@@ -2,8 +2,30 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
-## 2018 Q4
+## 2019 Q1
 
+- **The IPFS Org gets a fully dedicated Package Managers Working Group**
+  - `P0` - @daviddias - A Package Manager Working Group Roadmap is created
+  - `P0` - @??? - Onboard 2+ contributors to this Working Group       
+- **Support and guide the IPFSConf planning and Content Curation**
+  - `P0` - @daviddias - The IPFSConf is scheduled, announced and there is a CFP published.
+  - `P0` - @momack2 - The IPFS Org (Working Groups & Core Contributors) has a unified plan with goals for IPFSConf
+- **The IPFS Working Groups feel supported and they have the information they need to make the best decisions**
+  - `P1` - @parkan - Ship the first IPFS User Quarterly Report. It documents achievements and demos of new capabilities
```

Comment:
@parkan are you onboard with doing quarterly user reports? My understanding is this would include lots of user experience work over collab requests - so not sure if @meiqimichelle or I would make a better DRI for packaging and phrasing this from a product/UXR perspective.

Reply:
@parkan are you onboard to own this?